karmada-CLI-installtion-and-deploy-example/step1/verify.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+
+kubectl --kubeconfig=$HOME/.kube/config-member1 config get-contexts kind-member1
```

Comment:
Add a line break, same as in other places.

Reply:
this comment hasn't been addressed.

Reply:
ok